wireapp/wire-android
# chore: hide try again text [AR-2501]

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-2501" title="AR-2501" target="_blank"><img alt="Story" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10815?size=medium" />AR-2501</a>  Remove "Try again" button when a message fails to send
  </td></table>
  <br />
 




----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

try again when the message failed was not implemented yet 


### Solutions
 hide try again text till we implement the functionality of it

### Testing

#### How to Test
try to make your message to fail to send for any reason (hopefully you won't be able to test this 😅) , so you will see as showing in the ScreenShots below

### Attachments (Optional)

![try again before](https://user-images.githubusercontent.com/34056732/192093025-4f57b708-465e-4786-8f7b-7f3e74daa3ee.png)
![try again after](https://user-images.githubusercontent.com/34056732/192093023-d9ce8a0c-f3b1-48ce-82d9-ccd541dcd5eb.png)


----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
